extras/create_send_notifications.py

```diff
@@ -42,26 +42,41 @@ def create_output_directories():
         os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR)
     )
 
+def build_notifications_org_list(db):
+    """Return notifications organization list.
 
-def build_cyhy_org_list(db):
-    """Build list of CyHy organization IDs.
-
-    This is the list of CyHy organization IDs (and their descendants) that
-    receive CyHy reports.
+    This is the list of organization IDs that should
+    have a notification generated and sent.
+    """
+    notifications_to_generate = set()
+    cyhy_parent_ids = set()
+    ticket_owner_ids = db.notifications.distinct("ticket_owner")
```

Comment:
Would this work? I'm kind of loathe to hard code something that is managed elsewhere and I believe this is how we usually handle this kind of logic.

```suggestion
    ticket_owner_ids = db.NotificationDoc.collection.distinct("ticket_owner")
```

Reply:
You are correct @mcdonnnj - we should be doing it that way.